src/forms-opportunities/utils.js

```diff
@@ -385,7 +401,12 @@ export function getUrlsDataForAccessibilityAudit(scrapedData, context) {
             return;
           }
           if (sr.id) {
-            formSources.push(`form#${sr.id}`);
+            if (!addedFormSources.has(`form#${sr.id}`)) {
```

Comment:
not every form will have id.
we should handle this case for class also and write a generic logic.